katharsis-client/src/main/java/io/katharsis/client/ClientException.java

```diff
@@ -0,0 +1,29 @@
+package io.katharsis.client;
+
+import io.katharsis.errorhandling.ErrorData;
+import io.katharsis.errorhandling.exception.KatharsisMappableException;
+
+/**
+ * General client exception if no custom mapper is found.
+ */
+public class ClientException extends KatharsisMappableException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This class has 6 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)